climateconnect/climateconnect
# Feat: SocialMediaButton component

## Description

From our UI design https://xd.adobe.com/view/861b3ade-6eba-45d8-beb3-506e7afd7699-d348/screen/bfc077c5-f57c-4db1-b72b-79f69fb62544/ for the organization page, we wish to later add the ability to add social media links on organization page. 
This PR aims to make a generic social media button that can be used throughout the site e.g. in the footer.

## Test plan

1. Open a page where the footer is shown
2. Scroll down and see the footer
3. Icons should be behaving the same as before

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
